clients/client-python/tests/integration/test_gvfs_with_gcs_credential.py

```diff
@@ -0,0 +1,112 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import logging
+import os
+from random import randint
+import unittest
+
+from gcsfs import GCSFileSystem
+
+from gravitino import Catalog, Fileset, GravitinoClient
+from gravitino.filesystem import gvfs
+from tests.integration.test_gvfs_with_gcs import TestGvfsWithGCS
+
+logger = logging.getLogger(__name__)
+
+
+def gcs_with_credential_is_configured():
+    return all(
+        [
+            os.environ.get("GCS_STS_SERVICE_ACCOUNT_JSON_PATH") is not None,
+            os.environ.get("GCS_STS_BUCKET_NAME") is not None,
+        ]
+    )
+
+
+@unittest.skipUnless(gcs_with_credential_is_configured(), "GCS is not configured.")
+class TestGvfsWithGCSCredential(TestGvfsWithGCS):
```

Comment:
The current xxCredential tests depends on HDFS test class which couldn't be tested on Mac and should run with docker environment, it's not convenient, could you fix it in this PR or create separate issue to track it?

Reply:
ok, I will create a issue to track this problem as it's just a improvement.